lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -449,7 +505,14 @@ class AwsCompileEventBridgeEvents {
     return null;
   }
 
-  addInputConfigToTarget({ target, Input, InputPath, InputTransformer }) {
+  addInputConfigToTarget({
```

Comment:
This utility has confusing name now as it adds some more configuration than just `input`-related one to the targed. Could we change this name a little bit?